packages/create-pantheon-decoupled-kit/src/templates/partials/nextjs-shared/searchInputCSSModulesT.ts

```diff
@@ -0,0 +1,56 @@
+export const searchInputCSSModules = () => `.container {
```

Comment:
```suggestion
export const searchInputCSSModules = () => /* css */ `.container {
```
The lang comments (idk what to call these?) can be added to the partials as well.